app/modules/topics/saga/api/get.js

```diff
@@ -0,0 +1,53 @@
+// @flow
+
+import { call, put } from 'redux-saga/effects';
+
+import * as t from '../../actionTypes';
+
+import Api from '../../api';
+import type { Topic } from '../../model';
+
+import { addToState, setItemsInState } from '../../actions';
+
+export const apiGetTopicSaga = function* (action: t.GetAction): Generator<*, *, *> {
+  try {
+    const responseTopic = yield call(Api.get, action.payload.id);
+    const responseUser = yield call(Api.getUserId, action.payload.id);
+
+    const item = responseTopic.body.data;
+    const userId = responseUser.body.data.id;
```

Comment:
This isn't in our API documentation, but you can actually include related resources: `GET /topics/1?include=user` will yield an extra `included` section which contains the `user` relationship.